dmcp/fix.py

```diff
@@ -12,45 +13,59 @@
 from cvxpy.constraints.zero import Zero
 from cvxpy.constraints.psd import PSD
 
+from cvxpy.constraints.nonpos import NonNeg
+from cvxpy.constraints.nonpos import Inequality
+
 def fix(obj, fix_vars):
     """
     Fix the given variables in the object
     :param obj: a problem or an expression
     :param fix_var: a list of variables
     :return: a problem or an expression
     """
+    # Test whether fix_vars is already list, if not, recast
+    if not isinstance(fix_vars,list):
+       fix_vars = [fix_vars]
 
     # Create list of parameters
     variable_list = obj.variables()
     variable_list.sort(key = lambda x:x.id)
     param_list = []
-    for var in variable_list:
+    for var in variable_list:        
```

Comment:
use flake8 to detect whitespace/formatting issues.

Reply:
sg